Source/Core/VideoBackends/OGL/TextureConverter.cpp

```diff
@@ -258,11 +258,11 @@ static void EncodeToRamUsingShader(GLuint srcTexture,
 		glReadPixels(0, 0, (GLsizei)dstWidth, (GLsizei)dstHeight, GL_BGRA, GL_UNSIGNED_BYTE, nullptr);
 		u8* pbo = (u8*)glMapBufferRange(GL_PIXEL_PACK_BUFFER, 0, dstSize, GL_MAP_READ_BIT);
 
-		for (int i = 0; i < readLoops; i++)
+		for (int i = 0; i < dstHeight; i++)
 		{
-			memcpy(destAddr, pbo, readStride);
-			pbo += readStride;
-			destAddr += writeStride;
+			memcpy(destAddr, pbo, dstWidth*4);
+			pbo += dstWidth * 4;
+			destAddr += dstStride*2;
```

Comment:
I've never seen a "stride" in pixels. Can you move this *2 to the first place where this variable is called "stride"?